app/controllers/hub/faq_items_controller.rb

```diff
@@ -0,0 +1,69 @@
+module Hub
+  class FaqItemsController < ApplicationController
+    include AccessControllable
+    before_action :require_sign_in
+    before_action :require_admin
+    before_action :set_paper_trail_whodunnit
+    load_and_authorize_resource
+    layout "hub"
+
+    def new
+      @faq_category = FaqCategory.find(params[:faq_category_id])
+      faq_items = @faq_category.faq_items
+      @position_options = faq_items ? (1..(faq_items.count+1)).to_a : [1]
+    end
+
+    def create
+      slug = faq_item_params[:question_en].parameterize(separator: '_')
+      @faq_item = FaqItem.new(faq_item_params.merge(slug: slug))
+
+      if @faq_item.save
+        flash_message = "Successfully created '#{@faq_item.question_en}'"
+        render :show, notice: flash_message
+      else
+        flash_message = "Unable to create '#{@faq_item.question_en}', check validations"
+        render :new, error: flash_message
+      end
+    end
+
+    def edit
+      @position_options = FaqCategory.find(params[:faq_category_id]).faq_items.pluck(:position)
```

Comment:
add before action for @faq_category = FaqCategory.find(params[:faq_category_id])